app/models/stolen_record.rb

```diff
@@ -105,18 +105,20 @@ def address(skip_default_country: false, override_show_address: false)
     ].reject(&:blank?).join(", ")
   end
 
-  # Doesn't include street
-  def address_short
-    [city, state&.abbreviation, zipcode].reject(&:blank?).join(",")
```

Comment:
The only call sites for `StolenRecord#address_short` are in the v2 stolen record and bike serializers. This commit refactors to use `address_location` instead, which better accounts for international locations.